lib/Varien/Db/Adapter/Pdo/Mysql.php

```diff
@@ -1590,14 +1590,9 @@ public function loadDdlCache($tableCacheKey, $ddlType)
             return $this->_ddlCache[$ddlType][$tableCacheKey];
         }
 
-        if ($this->_cacheAdapter instanceof Zend_Cache_Core) {
+        if ($this->_cacheAdapter) {
```

Comment:
@justinbeaty actually I've just removed the check, cause I think the rest of the code should be executed, right?

Reply:
Yeah, it should because if not `null` then `$this->_cacheAdapter` should be an instance of `Mage_Core_Model_Cache`

Reply:
do you think we should have a check? it seems a bit redundant to me

Reply:
I don't think it's necessary since we have the PHP type checking.